scripts/docker_host_setup.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
```

Comment:
Since this is not docker specific, we can rename this file "configure_jetson.sh" or something.  

As a note (not necessarily related to this PR), but it may even be worth putting also putting these commands inline in the software setup as well.

1. Because they modify the system behavior, it's good to be transparent about what's happening

2. These are helpful Jetson commands for developers to learn

Reply:
That's true.
File renamed.